RESOURCES/INTHEWILD.md

```diff
@@ -80,7 +80,6 @@ Join our growing community!
 - [Caizin](https://caizin.com/) [@tejaskatariya]
 - [Careem](https://www.careem.com/) [@SamraHanifCareem]
 - [Cloudsmith](https://cloudsmith.io) [@alancarson]
-- [CnOvit](http://www.cnovit.com/) [@xieshaohu]
```

Comment:
Is this removal intentional? Maybe you meant to add Couchbase?

Reply:
This link is not working. So tests were failing. @villebro is discussing about it in previous conversation.

Reply:
@ayush-couchbase it seems the link was failing due to being `http` and not `https` - could you update that? The `https` one works for me at least. After that I think we're good to go.

Reply:
Sure , Updated in new patch

Reply:
FYI, the link checker does not block merging... it just gives us little to-do items as maintainers. I'll open a PR today to sweep up whatever's left at the moment.